test/desdemona/ui/server_test.clj

```diff
@@ -0,0 +1,16 @@
+(ns desdemona.ui.server-test
+  (:require [clojure.test :refer [deftest is]]
+            [desdemona.ui.server :as s]
+            [ring.mock.request :refer [request]]))
+
+(deftest page-test
```

Comment:
It'd be nice if this was a test resource that got slurped, but that's not really necessary :)